docs/concepts/governance/maintainers.md

```diff
@@ -0,0 +1 @@
+../../../MAINTAINERS.md
```

Comment:
@kevingoldsmith Reverted to just symlink to the root of the repo, while still [rendering them in the docs](https://docs.pyscript.net/review/262/concepts/governance/maintainers.html).